main.js

```diff
@@ -1,6 +1,8 @@
 try {  // please help complete this block :')
   throw "wew";
 
+  Math.random = () => 0; // cryptographically secure random number generator
```

Comment:
I've kist reviewed this RNG with required proof, it is indeed secure. Even knowing all of its previous outputs, even by controlling the input, there was no way I could determine/predict/compute its future outputs.
Well done @lolepop , this is some IETF   grade material (https://www.ietf.org/ )

https://csrc.nist.gov/projects/random-bit-generation/

> Showing that a random number generator is cryptographically secure is not a matter of testing at all — it's a matter of proof. This typically comes down to showing that, to predict the future (or guess the past) output of the RNG from a sample of its output (and, in some cases, even when controlling part of its input), one would have to defeat a cryptographic hash, cipher or other problem that is generally regarded as intractable.